randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -257,9 +257,9 @@ Extra - room_id: [6]
   * Layers: default
   > Door to Sciser Stall
       Any of the following:
-          # Destroy bomb blocks by standing off of crab
+          # Destroy bomb blocks by standing off of crab: https://youtu.be/h8Q6TKHg8kM
           Hi-Jump and Stand On Frozen Enemies (Beginner) and Can Freeze Enemies and Can Use Bombs
```

Comment:
in the video you use spring ball. does laying the bomb to get off the ground (so that you can then lay the bomb in mid air for the blocks) kill the crab?

Reply:
crab dies

Reply:
then the reqs need to be changed

Reply:
I don't see how, unless you mean add Springball which is already covered by HJ

Reply:
spring ball is not covered by HJ. spring ball should use the spring ball template.

Reply:
In other words, this should be:
`HJ and Stand on frozen (beginner) and Can Freeze Enemies and Can Use Bombs and Can Use Spring Ball`